org.gnome.Snapshot.json

```diff
@@ -0,0 +1,212 @@
+{
+    "id": "org.gnome.Snapshot",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "44",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "snapshot",
+    "finish-args": [
+        "--device=dri",
+        "--filesystem=xdg-pictures/Camera:create",
+        "--filesystem=xdg-run/pipewire-0",
+        "--filesystem=xdg-videos/Camera:create",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=pulseaudio",
+        "--socket=wayland"
+    ],
+    "build-options": {
+        "append-path": "/usr/lib/sdk/rust-stable/bin"
+    },
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+	"/share/installed-tests",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "pipewire",
```

Comment:
what's the rationale for building pipewire?

Reply:
I was told by Maximilliano that 0.3.69 was not in the current 44 runtime, are we mistaken?

Reply:
$ pkg-config --modversion libpipewire-0.3
0.3.69

Reply:
Hm, ok then. I'll remove it

Reply:
The runtime added .69 mere days ago, anyone who installs the app without running an update in the previous few days is going to have a borked app and no way of knowing what went wrong. 

We will remove it in 2-3 weeks, but for now i would rather have it.